[CI] SumLongAggregatorFunctionTests testOverflowFails failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #12448 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/w5shtqfpeby44)
- [elasticsearch-periodic-platform-support #12448 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/46x2bqk7ecpsg)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:compute:test" --tests "org.elasticsearch.compute.aggregation.SumLongAggregatorFunctionTests.testOverflowFails" -Dtests.seed=1736D1DC5EB756A6 -Dtests.locale=kam-Latn-KE -Dtests.timezone=US/Aleutian -Druntime.java=26
```

**Applicable branches:**
9.4

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.compute.aggregation.SumLongAggregatorFunctionTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testOverflowFails'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: iterable containing [a string containing "\"Line 1:1: evaluation of [source] failed, treating result as null. Only first 20 failures recorded.\"", a string containing "\"Line 1:1: java.lang.ArithmeticException:"]
     but: no item was a string containing "\"Line 1:1: evaluation of [source] failed, treating result as null. Only first 20 failures recorded.\""
```

**Issue Reasons:**
- [9.4] 2 consecutive failures in test testOverflowFails
- [9.4] 2 consecutive failures in step amazonlinux-2023_platform-support-aws
- [9.4] 1 consecutive failures in pipeline elasticsearch-periodic-platform-support
- [9.4] 2 failures in test testOverflowFails (5.9% fail rate in 34 executions)
- [9.4] 2 failures in step amazonlinux-2023_platform-support-aws (100.0% fail rate in 2 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -214,9 +214,6 @@ tests:
 - class: org.elasticsearch.upgrades.TransformSurvivesUpgradeIT
   method: testTransformRollingUpgrade
   issue: https://github.com/elastic/elasticsearch/issues/150442
-- class: org.elasticsearch.compute.aggregation.SumLongAggregatorFunctionTests
-  method: testOverflowFails
-  issue: https://github.com/elastic/elasticsearch/issues/150455
 - class: org.elasticsearch.xpack.stateless.recovery.BlockRefreshUponIndexCreationIT
   method: testRefreshesAreBlockedUntilBlockIsRemoved
   issue: https://github.com/elastic/elasticsearch/issues/150492
@@ -636,9 +633,6 @@ tests:
 - class: org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT
   method: test {yaml=downsample/100_downsample_histograms/Downsample tdigest with last value}
   issue: https://github.com/elastic/elasticsearch/issues/155028
-- class: org.elasticsearch.compute.aggregation.SumLongGroupingAggregatorFunctionTests
-  method: testOverflowInGroupingProducesNullAndWarning
-  issue: https://github.com/elastic/elasticsearch/issues/150769
 - class: org.elasticsearch.indices.ShardLimitValidatorTests
   method: testValidateShardLimitOpenIndices
   issue: https://github.com/elastic/elasticsearch/issues/154859

--- a/x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/SumLongAggregatorFunctionTests.java
+++ b/x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/SumLongAggregatorFunctionTests.java
@@ -77,7 +77,7 @@ public class SumLongAggregatorFunctionTests extends AggregatorFunctionTestCase {
                 }
             )
         ) {
-            new TestDriverRunner().run(driver);
+            new TestDriverRunner().numThreads(1).run(driver);
         }
 
         assertDriverContext(driverContext);

--- a/x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/SumLongGroupingAggregatorFunctionTests.java
+++ b/x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/SumLongGroupingAggregatorFunctionTests.java
@@ -98,7 +98,7 @@ public class SumLongGroupingAggregatorFunctionTests extends GroupingAggregatorFu
                 () -> warnings.addAll(threadContext.getResponseHeaders().getOrDefault("Warning", List.of()))
             )
         ) {
-            new TestDriverRunner().run(driver);
+            new TestDriverRunner().numThreads(1).run(driver);
         }
 
         assertDriverContext(driverContext);

